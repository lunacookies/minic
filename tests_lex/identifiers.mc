foo
FOO
a
abc123
a_b
a_
_a
_123
