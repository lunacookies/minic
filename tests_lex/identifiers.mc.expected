{
	IDENTIFIER 0..3
	IDENTIFIER 4..7
	IDENTIFIER 8..9
	IDENTIFIER 10..16
	IDENTIFIER 17..20
	IDENTIFIER 21..23
	IDENTIFIER 24..26
	IDENTIFIER 27..31
}
